docker/rpc/Dockerfile

```diff
@@ -61,10 +61,11 @@ RUN build_platform() { \
     fi; \
     echo "Using BINDGEN_EXTRA_CLANG_ARGS: $BINDGEN_EXTRA_CLANG_ARGS"; \
     # Build the Rust application for the specified target
-    BINDGEN_EXTRA_CLANG_ARGS=$BINDGEN_EXTRA_CLANG_ARGS cargo build --all --release \
+    BINDGEN_EXTRA_CLANG_ARGS=$BINDGEN_EXTRA_CLANG_ARGS cargo build --all --all-features --release \
```

Comment:
will all features build in "testing" mode?

Reply:
unfortunatly yes :/ could maybe build a second time in debug mode with testing feature?

Reply:
isn't this bad for us returning the eth hash?